Gemfile.lock

```diff
@@ -82,7 +82,7 @@ GEM
       activesupport (>= 3.0.0, < 6.1)
       ruby2_keywords (>= 0.0.2, < 1.0)
     ast (2.4.1)
-    autoprefixer-rails (10.0.1.3)
+    autoprefixer-rails (10.0.3.0)
```

Comment:
I don't think we want to modify the .lock file for the same reason as above. It could cause issues with the deployment that's setup to run with Ruby 2.6.6